onnxruntime/hosting/protobuf/error_code.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
This file is unused. What's the purpose for it? I thought errors are always returned as JSON?

Reply:
Not sure why Klein left it. I've remove it for now.